README.md

```diff
@@ -6,7 +6,23 @@ This repository holds the kcp tests that tests against the publicly available  i
 * Golang installed. See [Installing Golang](https://golang.org/doc/install),the newer the better.
         * Ensure you install Golang from a binary release [found here](https://golang.org/dl/), not with a package manager such as `dnf`
 * golint installed. See [Installing golint](https://github.com/golang/lint#installation)
-* Have the environment variable `KUBECONFIG` set pointing to your cluster
+* Install [kubelogin](https://github.com/int128/kubelogin.git) and [KCP kubectl plugin](https://github.com/kcp-dev/kcp.git)
+      * Ensure your installed KCP kubectl plugin version is the same with the kcp service version
+  ```
+  $ git clone https://github.com/kcp-dev/kcp.git
+  # E.g. Install the release-0.5 version KCP kubectl plugin
+  # commitID: "fccfb61" of release-0.5 branch/tag
+  $ git checkout fccfb61
+  $ make install WHAT=./cmd/kubectl-kcp && make install WHAT=./cmd/kubectl-workspaces && make install WHAT=./cmd/kubectl-ws
+  ```
+
+* Have the environment variable `KUBECONFIG` set pointing to your kcp service
+* Login the kcp service via SSO (Single Sign On)
+  ```
+  kubectl oidc-login get-token --oidc-issuer-url=https://sso.xxxx.com/auth
+   --oidc-client-id=cloud-services --oidc-redirect-url-hostname=127.0.0.1
+  ```
+* Have the environment variable `PCLUSTER_KUBECONFIG` set pointing to your BYO cluster(Bring your own cluster), it's optional, you should set it if your test cases need to operate BYO cluster. The test cases which need to operate BYO cluster will be skipped by no environment variable `PCLUSTER_KUBECONFIG` found without setting it.
```

Comment:
@Phaow can we have this modified something like "If you would like to test BYO cases please make sure to set the PCLUSTER_KUBECONFIG else all the tests related to BYO will be skipped" ?

Reply:
Sure. I'll update it.

Reply:
Addressed.